staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_test.go

```diff
@@ -206,6 +206,12 @@ func configFromEnvOrSim() (VSphereConfig, func()) {
 	return configFromSim()
 }
 
+func TestMain(m *testing.M) {
+	klog.Flush()
+	klog.InitFlags(nil)
```

Comment:
`klog.InitFlags(nil)` can go into an `init` function. Then you don't need this `TestMain` at all.

Reply:
The `vsphere_test` is in package `vsphere`, if i use `init` here i'm afraid code can't define another `init` function for real logic init.

Reply:
The `*_test.go` files are not getting compiled into the `vsphere` package so there's no risk of polutting the package.

You can also have more than one `init` function, even in the same file.

Reply:
You just have to ensure that there is exactly one `init` function which calls `klog.InitFlags(nil)`.

Reply:
Thanks for explanation!

Reply:
Done